src/constants/api.ts

```diff
@@ -0,0 +1,3 @@
+export const ExplorerEndpoints = {
```

Comment:
@celine Since you already did it, I think I could remove this. When do you think you'll merge your PR?

Reply:
It's merged, feel free to rebase.